pkgs/applications/audio/airwave/default.nix

```diff
@@ -42,7 +41,7 @@ multiStdenv.mkDerivation {
 
   src = airwave-src;
 
-  nativeBuildInputs = [ cmake makeWrapper ];
+  nativeBuildInputs = [ cmake makeWrapper qt5.wrapQtAppsHook ];
 
   buildInputs = [ file libX11 qt5.qtbase wine-xembed ];
```

Comment:
we should never reference `qt5` directly, and instead should use the `libsForQt5.callPackage` function to bring all attributes within `qt5` into scope.

Reply:
see https://nixos.org/nixpkgs/manual/#sec-language-qt